quantuminspire/cli/command_list.py

```diff
@@ -334,6 +335,10 @@ def get_final_results(job_id: int = typer.Argument(..., help="The id of the run"
 @app.command("login")
 def login(
     host: Optional[str] = typer.Argument(None, help="The URL of the platform to which to connect"),
+    use_local_auth_config: bool = typer.Option(
```

Comment:
I do know that the Zen says "Explicit is better than implicit.", but it also says "Readability counts". I think this is a very length flag. We don't need the `use_` right? or maybe `override_auth_config`?

Reply:
I agree I like `override_auth_config` better